lib/components/molecules/Accordion/tests/Accordion.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { mount } from 'enzyme';
+// eslint-disable-next-line import/named
+import { defaultAccordion } from '../Accordion.mock';
+import Accordion from '../index';
+
+describe('<Accordion />', () => {
```

Comment:
These tests are just not enough, They are not even testing for small components that accordion uses.